initrd/init

```diff
@@ -76,7 +76,24 @@ else
 	DEBUG "Debug output enabled from /etc/config.user's CONFIG_DEBUG_OUTPUT=y after combine_configs (Config menu enabled Debug)"
 fi
 
-TRACE "Under init"
+TRACE_FUNC
```

Comment:
I think this has to be reverted to `TRACE`, or we have to drop legacy-flash boards and use bash for this script.

`TRACE_FUNC` isn't available in ash_functions for ash scripts, it's in functions and uses bashisms.  (I'm getting an error on the console at boot on line 79.)

Reply:
@JonathonHall-Purism fixed. As for deprecating legacy-flash boarss, I think we will reach limits of space there really soon, so deprecation will happen there. Could open an issue for that to reprioritize. Woud love to have those TRACE_FUNC everywhere, but not related here.